.env.sample

```diff
@@ -1,3 +1,4 @@
 # These environment variables are required to run the application locally
 
-SENTRY_DSN:this-is-where-you-enter-the-sentry-dsn
+SENTRY_DSN=this-is-where-you-enter-the-sentry-dsn
+CFE_HOST=enter-a-vaid-url-here-or-http://localhost:3000
```

Comment:
IMHO this is a poor default - CFE won't be running locally, and I don't think we want to always run EFE + CFE + (the other API) on our dev machines.

Reply:
I have the following in my local .env file:
CFE_HOST=https://check-financial-eligibility-staging.cloud-platform.service.justice.gov.uk
#CFE_HOST=http://localhost:3002
So I can easily switch between staging and local. When developing on both of these services we may want to have both running locally. Similarly this is where you would enter the uat branch for CFE e.g ap-3415-update-uat-postgres-check-financial-uat.cloud-platform.service.justice.gov.uk if you want to test local EFE against a CFE UAT branch.

Reply:
I agree. I just don't think it's a very good default. I don't have the configuration that you have, and hopefully we won't need this for local (as we have the Legal Framework API to deal with as well). IMHO the 'out of the box' setup should be simple and 'just work' i.e. 'rake spec' runs without having to know the address of the staging CFE instance)